lib/zone.ts

```diff
@@ -564,6 +571,27 @@ const Zone: ZoneType = (function(global: any) {
       }
     }
 
+    static get rootZone(): AmbientZone {
+      let zone = Zone.current;
+      while (zone.parent) {
+        zone = zone.parent;
+      }
+      return zone;
+    }
+
+    static runOutsideOfCurrentZone(
```

Comment:
I think this API gives too much complexity to the user. Same can be achieved by:
```
Zone.root.fork(zoneSpec).run(....)
```

So I don't see the need to have it.